src/mixed-line-bar-chart/chart-container.tsx

```diff
@@ -349,12 +352,6 @@ export default function ChartContainer<T extends ChartDataTypes>({
     return () => document.removeEventListener('keydown', onKeyDown);
   }, [dismissPopover]);
 
-  useLayoutEffect(() => {
-    if (highlightedX !== null || highlightedPoint !== null) {
-      showPopover();
-    }
-  }, [highlightedX, highlightedPoint, showPopover]);
```

Comment:
If highlightedX was `1` (showing the popover for the 1st group) and then it changes as `1` -> `null` -> `1` in a quick succession, that intermediate transition to `null` is ignored by React 18. As result, the effect callback is not fired, thus not showing the popover.